### PR TITLE
feat(dedicated-cloud.user): add encryptionRight field

### PR DESCRIPTION
**Pull request by carsso**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | `PUPCC-21824`
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

PCC : Adding new field allowing user to manage encryption

In user table : 
Capture d’écran 2022-07-13 à 16 36 23

In user edition form : 
Capture d’écran 2022-07-13 à 16 36 28

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/dedicatedCloud-user.html [hunk 1]
```diff
@@ -165,6 +165,14 @@
             <span data-translate="common_no" data-ng-if="!$row.nsxRight">
             </span>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'dedicatedCloud_USER_encryption' | translate"
+        >
+            <span data-translate="common_yes" data-ng-if="$row.encryptionRight">
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/dedicatedCloud-user.html:171]: Would it be possible to add this translation here https://github.com/ovh/manager/blob/ec7b96a67adc8a61d1158f8b7f22cb9baef2ad65/packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/translations/Messages_fr_FR.json instead  of using commons ?

Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/dedicatedCloud-user.html:171]: Sure, will do.
I am not really aware of your coding rules so I just checked how you did for other fields nearby and did the same.

Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/dedicatedCloud-user.html]: Done and enerything is still working (but will require more translations)
Capture d’écran 2022-07-19 à 14 54 53

**Review by marie-j — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by vks-ovh — DISMISSED** [review]

**Comment by marie-j** [comment]
Hello @carsso could you please target `master` branch instead ? 
Thanks

**Comment by carsso** [comment]
@marie-j Done :)

**Pull request merged**